### PR TITLE
New branch for mobile launch [merging 10/10]

### PR DESCRIPTION
**Pull request by brucejlin1**

### CODE REVIEW

File: _mobile/home.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+---
+
+# PyTorch Mobile
+
+Running ML on edge devices is growing in importance as applications continue to demand lower latency. It is also a foundational element for privacy-preserving techniques such as federated learning. As of PyTorch 1.3, PyTorch supports an end-to-end workflow from Python to deployment on iOS and Android. 
+
+This is an early, experimental release that we will be building on in several areas over the coming months:
+
+- Extends the PyTorch API to cover common preprocessing and integration tasks needed for incorporating ML in mobile applications
+- Support for FBGEMM and QNNPACK quantized kernel libraries support for x86 and ARM CPUs respectively
```
Comment (dzhulgakov) [inline _mobile/home.md]: why is FBGEMM here? do we have x86 embedded devices?

File: _mobile/home.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+published: true
+---
+
+# PyTorch Mobile
+
+Running ML on edge devices is growing in importance as applications continue to demand lower latency. It is also a foundational element for privacy-preserving techniques such as federated learning. As of PyTorch 1.3, PyTorch supports an end-to-end workflow from Python to deployment on iOS and Android. 
+
+This is an early, experimental release that we will be building on in several areas over the coming months:
+
+- Extends the PyTorch API to cover common preprocessing and integration tasks needed for incorporating ML in mobile applications
```
Comment (dzhulgakov) [inline _mobile/home.md]: extend (without 's')

And I'd maybe formulate it as "Provide APIs to cover common..." as it won't be part of PT core (more like torchvision for mobile)

File: _mobile/home.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+- Support for FBGEMM and QNNPACK quantized kernel libraries support for x86 and ARM CPUs respectively
+- (Coming soon) Build level optimization and selective compilation depending on the operators needed for user applications (i.e., you pay binary size for only the operators you need)
```
Comment (dzhulgakov) [inline _mobile/home.md]: this entire list is Coming soon, so I'd drop it here

File: _mobile/android.md [hunk 4]
```diff
@@ -0,0 +1,282 @@
...
+# Android
+
+## Quickstart with a HelloWorld Example
+
+[HelloWorld](https://github.com/pytorch/android-demo-app/tree/master/HelloWorldApp) is a simple image classification application that demonstrates how to use PyTorch Android API.
+This application runs TorchScript serialized TorchVision pretrained resnet18 model on static image which is packaged inside the app as android asset.
+
+#### 1. Model Preparation
+
+Let’s start with model preparation. If you are familiar with PyTorch, you probably should already know how to train and save your model. In case you don’t, we are going to use a pre-trained image classification model(Resnet18), which is packaged in [TorchVision](https://pytorch.org/docs/stable/torchvision/index.html).
```
Comment (dzhulgakov) [inline _mobile/android.md]: looks like link got lost - copy from the iOS tutorial:

`[Resnet18](https://pytorch.org/hub/pytorch_vision_resnet/)`

File: _features/mobile.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    ## iOS prebuilt binary
+    pod ‘LibTorch’
+    ## Android prebuilt binary
+    implementation 'org.pytorch:pytorch_android:1.3.0'
+
+    ## Run your model (Android example)
+    Tensor input = Tensor.fromBlob(data, new long[]{1, data.length});
+    IValue output = module.forward(IValue.tensor(input));
+    float[] scores = output.getTensor().getDataAsFloatArray();
+
```
Comment (xta0) [inline _features/mobile.md]: nit: delete this blank line?

File: _mobile/home.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+- (Coming soon) Further improvements to performance and coverage on mobile CPUs and GPUs
+
+Learn more or get started on [Android](http://pytorch.org/mobile/android) or [iOS](http://pytorch.org/mobile/ios).
```
Comment (xta0) [inline _mobile/home.md]: This should be `{{site.baseurl}}/mobile/android`, `{{site.baseurl}}/mobile/ios`

File: _get_started/mobile.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+## Get Started with PyTorch Mobile
+
+As of PyTorch 1.3, PyTorch supports an end-to-end workflow from Python to deployment on iOS and Android. 
+This is an early, experimental release that we will be building on in several areas over the coming months.
+
+Get started on [Android](https://pytorch.org/mobile/android)
+
+Get started on [iOS](https://pytorch.org/mobile/ios)
+
```
Comment (xta0) [inline _get_started/mobile.md]: `{{ site.baseurl }}/mobile/android`, `{{ site.baseurl }}/mobile/ios`

File: _features/mobile.md (hunk 5 above)
Comment (brucejlin1) [reply _features/mobile.md]: Deleted, thank you

File: _mobile/home.md (hunk 1 above)
Comment (jspisak) [reply _mobile/home.md]: We should remove FBGEMM and the mention of x86. Good catch @dzhulgakov 

cc @brucejlin1

File: _mobile/home.md (hunk 2 above)
Comment (jspisak) [reply _mobile/home.md]: Sounds good. 

Let's go with:

- Provides APIs that cover common preprocessing...

File: _mobile/home.md (hunk 3 above)
Comment (jspisak) [reply _mobile/home.md]: +1 - let's remove for now..

Comment (jspisak) [reply _mobile/home.md]: Yes, let's remove for now..  cc @brucejlin1

File: _mobile/home.md (hunk 1 above)
Comment (brucejlin1) [reply _mobile/home.md]: Removed

File: _mobile/home.md (hunk 2 above)
Comment (brucejlin1) [reply _mobile/home.md]: Updated

File: _mobile/home.md (hunk 3 above)
Comment (brucejlin1) [reply _mobile/home.md]: Removed "Coming soon" text

File: _get_started/mobile.md (hunk 7 above)
Comment (brucejlin1) [reply _get_started/mobile.md]: Fixed links, thank you

**Pull request merged**